docs/se/grpc/09_marshalling.adoc

```diff
@@ -32,14 +32,6 @@ Protobuf marshaller will be used by default for any request and response classes
 
 That means that you don't need any special handling or configuration in order to support Protobuf serialization of requests and responses.
 
-=== Java Serialization Marshalling
```

Comment:
Good catch. I will remove Java Serialization from `Default Marshalling Support` section. The JsonBMarshaller is already mentioned in the `Custom Marshalling` section so don't need to mention this under `Default Marshalling Support` section.

Reply:
Since Default Marshalling only has one option now, i.e. Photobuf, I will remove it as a subsection and instead integrate it under the `Default Marshalling Support` section similar to `Custom Marshalling` section containing description about `JsonbMarshaller`